include/utility.hpp

```diff
@@ -569,37 +546,19 @@ namespace giml {
             }
         }
         // Copy assignment operator
-        LinkedList<T>& operator=(const giml::LinkedList<T>& l) {
-
-        }
-        //Destructor
+        LinkedList<T>& operator=(const giml::LinkedList<T>& l) {}
```

Comment:
Why is this empty? Either purposefully `delete` it or define it.

Reply:
This one's yours lol do what u want